tests/optimize/test_objective.py

```diff
@@ -74,5 +74,27 @@ def tearDown(self):
         self.directory.cleanup()
         del self.directory
 
+class test_RelativeEntropy(unittest.TestCase):
+    """Unit tests for relentless.optimize.RelativeEntropy"""
+
+    def test_compute(self):
+        """Test compute method"""
+        v_obj = relentless.volume.Cube(L=3.0)
+        target = relentless.ensemble.Ensemble(T=10, V=v_obj, N={'A':1, 'B':2})
+
+        thermo = relentless.simulate.dilute.AddEnsembleAnalyzer()
+        operations = [relentless.simulate.InitializeRandomly(seed=2,neighbor_buffer=0.4),
+                      relentless.simulate.AddNVTIntegrator(dt=0.1,tau_T=1.0),
+                      thermo]
+        simulation = relentless.simulate.dilute.Dilute(operations)
+
+        lj = relentless.potential.LennardJones(types=('1','2'))
```

Comment:
You need to define the parameters and design variables for the lj potential here. I'm surprised this even runs, actually... do the potentials not raise an error if one of their parameters is not set?

Also, make life easier on yourself and start with just one type for now.